lib/api/slack.ex

```diff
@@ -16,12 +18,44 @@ defmodule Aprb.Api.Slack do
       requires :response_url, type: String
     end
     post do
+      # check that token matches, that the POST comes from our slack integration
+      if System.get_env("SLACK_SLASH_COMMAND_TOKEN") != params[:token] do
+        conn
+          |> put_status(403)
+          |> json(%{message: "Unauthorized"})
+          |> halt()
+      end
 
-      # TODO: check that token matches, that the POST comes from our slack integration
-      # if System.get_env("SLACK_SLASH_COMMAND_TOKEN") != params["token"] ...
+      subscriber = case Repo.get_by(Subscriber, user_id: params[:user_id]) do
```

Comment:
If I understand this correctly every user will have their own subscription, so we'll get duplicates if two different users subscribe to the same thing in the same channel. I think we don't need to know anything about the subscribing user, so the subscription should just be a channel + subscription.